errdefs/grpc.go

```diff
@@ -104,9 +104,9 @@ func FromGRPC(err error) error {
 
 	msg := rebaseMessage(cls, err)
 	if msg != "" {
-		err = errors.Wrap(cls, msg)
+		err = fmt.Errorf("%s: %w", msg, cls)
 	} else {
-		err = errors.WithStack(cls)
+		err = cls
```

Comment:
is it changing the behavior?

Reply:
by @dmcgowan comment https://github.com/containerd/containerd/pull/6366#issuecomment-992018952. i prefer directly returning the error